requirements.txt

```diff
@@ -1,5 +1,5 @@
 PyYAML==3.10
-docker-py==1.2.2
+git+https://github.com/lukemarsden/docker-py.git@e2ba8a54935df8d40676529f67b16f5927e51000#egg=docker-py
```

Comment:
Just noticed we almost certainly don't want to merge this until this has been replaced with a stable release of `docker-py` which includes https://github.com/docker/docker-py/pull/628